### ISSUE TITLE
init scripts

### ISSUE THREAD

**Issue by OlofBlomqvist** [OlofBlomqvist/odd-box#43]
It would be nice to have a way of configuring a script/command to run before starting a site.
For example, one could do git pull && rebuild so that the site is always the latest version?

**Comment by OlofBlomqvist** [comment]
.. I think we will delay this for now. It's already possible to just create a custom script that does init logic and then runs the binary, and use that as the entrypoint instead.
